### PR TITLE
Add Ginger Colors, Typography etc.

### PR DESCRIPTION
**Pull request by jcarver989**
This PR adds branding boilerplate, taken from ginger.io/brand.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```
Comment (jcarver989) [inline _layouts/default.html:1]: Very basic layout. We'll want something fancier that shows all the blog posts on a "home" page and a way to click into an individual post.

File: _sass/colors.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// From: https://www.ginger.io/identity-system#color
```
Comment (jcarver989) [inline _sass/colors.scss:1]: Colors

File: _sass/nav.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@import "colors.scss";
```
Comment (jcarver989) [inline _sass/nav.scss:1]: Basic header... doesn't really need to be there but figured it'd be a good example. 

image

File: _sass/typography.scss [hunk 4]
```diff
@@ -0,0 +1,60 @@
+/** Styles and fonts copied from https://www.ginger.io/brand */
```
Comment (jcarver989) [inline _sass/typography.scss:1]: Typography.

File: _sass/typography.scss [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+@font-face {
+  font-family: "Heldane Display";
+  src: url(/fonts/Heldane_Display_Regular.otf);
```
Comment (jcarver989) [inline _sass/typography.scss:5]: ginger.io/brand loads these, so grabbed them from there

File: assets/css/main.scss [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---
+---
+
+@import "colors.scss";
+@import "typography.scss";
+@import "nav.scss";
+
+* {
+  box-sizing: border-box;
```
Comment (jcarver989) [inline assets/css/main.scss:10]: This basically makes CSS sizing sane for people not accustomed to width/height calculations involving padding/margins.

File: assets/css/main.scss [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  -webkit-font-smoothing: antialiased;
```
Comment (jcarver989) [inline assets/css/main.scss:11]: And these just make typography look a bit nicer on osx

File: _sass/typography.scss [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+h5,
+h6 {
+  font-family: "Heldane display", sans-serif;
+}
+
+h1 {
+  margin: 0.67em 0;
+  margin-top: 0px;
+  margin-bottom: 25px;
+  font-size: 96px;
```
Comment (jcarver989) [inline _sass/typography.scss]: We'll likely want to make the font size smaller...headers are HUUUUGE on the branding website :).

File: .vscode/settings.json [hunk 9]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (gbryan) [inline .vscode/settings.json:1]: Nitpick: probably don't want to check in this VSCode file

**Review by gbryan — APPROVED** [review]
Looks good!

File: .vscode/settings.json (hunk 9 above)
Comment (jcarver989) [reply .vscode/settings.json:1]: Actually this is a project configuration file for VSCode. I generally like to check these in as it ensures anybody using VSCode has the same settings applied to the project automatically.

I'm assuming more than just me uses VSCode...if that's not the case I'm happy to remove :)

Comment (gbryan) [reply .vscode/settings.json:1]: Ah ok cool! Didn't realize it was intentional. Sounds good to me.

**Review by aboisvert — APPROVED** [review]

**Pull request merged**